mbuild/packing.py

```diff
@@ -640,7 +662,17 @@ def _packmol_error(out, err):
 
 
 def _run_packmol(input_text, filled_xyz, temp_file):
+    """Apply PACKMOL packing based on the input text.
```

Comment:
```suggestion
    """Call PACKMOL to pack system based on the input text.
```
Just a little bit more explicit